client/Contexts/ConfigFeatureFlag/ConfigFeatureFlag.spec.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import {
+  ConfigFeatureFlagProvider,
+  ConfigFeatureFlagConsumer,
+} from './Context';
+
+describe('ConfigFeatureFlag context', () => {
+  describe('initial state tests', () => {
+    it('returns the expected configuration values', () => {
+      const { getByText } = render(
+        <ConfigFeatureFlagProvider>
```

Comment:
Did you find this was easier than using the provider helper?

Reply:
The `ConfigFeatureFlagProvider` isnt a regular provider - it is a component which makes a query that provides a value - thus I had to drive it as such. As you will see in #99 though, I do use your helper for anything which consumes only :)